cmd/nsm-init/nsm-init.go

```diff
@@ -39,8 +47,15 @@ const (
 var (
 	clientSocketPath     = path.Join(nsmdp.SocketBaseDir, nsmdp.ServerSock)
 	clientSocketUserPath = flag.String("nsm-socket", "", "Location of NSM process client access socket")
+	configMapName        = flag.String("configmap-name", "", "Name of a ConfigMap with requested configuration.")
+	kubeconfig           = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file. Either this or master needs to be set if the provisioner is being run out of cluster.")
```

Comment:
How are we injecting the kubeconfig file into the Pod?

Reply:
kubeconfig is only need for out of cluster case, when you run/debug init container as a standalone binary. For in-cluster case, it will get token automatically.

Reply:
@sbezverk Awesome!